lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
There isn't really any documentation. It's on its way to becoming a more general purpose Dart binding for Magenta syscalls (like our C++ libmx).

This set of changes the representation of magenta handles in Dart from an int to an [object](https://fuchsia-review.googlesource.com/c/49681/10/dart/sdk_ext/src/handle.dart#7) with a [native peer](https://fuchsia-review.googlesource.com/c/49681/10/dart/sdk_ext/src/handle.h#24).

This solves a couple of problems - it makes it easier to tie handles to particular Dart or Flutter applications running in a single runner, and allows us to manage the relationship between handle waiters and their handles better.